File: elearning-service/src/main/java/com/app/elearningservice/dto/EnrollCourseDTO.java
package com.app.elearningservice.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class EnrollCourseDTO {
    private String courseName;
    private String thumbnail;
    private String startDate;
    private String endDate;
    private String enrollmentDate;
    private String enrollmentStatus;
    private String courseStatus;
}

File: elearning-ui/src/app/shared/model/EnrollCourseDTO.ts
export class EnrollCourseDTO {
  constructor(
    public courseName: string = '',
    public thumbnail: string = '',
    public startDate: string = '',
    public endDate: string = '',
    public enrollmentDate: string = '',
    public enrollmentStatus: string = '',
    public courseStatus: string = '',
  ) {
  }
}
